vmdb/app/models/condition.rb

```diff
@@ -29,7 +29,7 @@ def applies_to?(rec, inputs={})
   end
 
   def self.conditions
-    find(:all).collect {|c| c.expression}
+    pluck(:expression)
```

Comment:
Yes, it looks like pluck handles it for us.  See below...

Note, this `["expression", "--- !ruby/string:VimString\nstr: b\nxsiType: \nvimType: \n"]` in the insert statement below:

```ruby
irb(main):008:0> Condition.create!(:description => "aaa", :modifier => "deny", :expression => VimString.new("b"), :towhat => "b")
...
  SQL (0.3ms)  INSERT INTO "conditions" ("applies_to_exp", "created_on", "description", "expression", "file_mtime", "filename", "guid", "miq_policy_id", "modifier", "name", "notes", "towhat", "updated_on") VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13) RETURNING "id"  [["applies_to_exp", nil], ["created_on", Thu, 15 Jan 2015 20:05:07 UTC +00:00], ["description", "aaa"], ["expression", "--- !ruby/string:VimString\nstr: b\nxsiType: \nvimType: \n"], ["file_mtime", nil], ["filename", nil], ["guid", "cd7614ec-9cf1-11e4-9d2e-3c15c2c9babc"], ["miq_policy_id", nil], ["modifier", "deny"], ["name", "cd7614ec-9cf1-11e4-9d2e-3c15c2c9babc"], ["notes", nil], ["towhat", "b"], ["updated_on", Thu, 15 Jan 2015 20:05:07 UTC +00:00]]
   (0.8ms)  COMMIT
=> #<Condition id: 4, name: "cd7614ec-9cf1-11e4-9d2e-3c15c2c9babc", description: "aaa", modifier: "deny", expression: "b", towhat: "b", file_mtime: nil, created_on: "2015-01-15 20:05:07", updated_on: "2015-01-15 20:05:07", guid: "cd7614ec-9cf1-11e4-9d2e-3c15c2c9babc", filename: nil, applies_to_exp: nil, miq_policy_id: nil, notes: nil>
irb(main):009:0> Condition.conditions
  SQL (0.3ms)  SELECT "conditions"."expression" FROM "conditions" WHERE ("conditions".id >= 0 AND "conditions".id <= 999999999999)
=> ["b"]
irb(main):010:0> Condition.conditions.first.class
  SQL (0.4ms)  SELECT "conditions"."expression" FROM "conditions" WHERE ("conditions".id >= 0 AND "conditions".id <= 999999999999)
=> VimString
```